PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/PrebidMobile.java

```diff
@@ -342,6 +327,27 @@ public static void checkGoogleMobileAdsCompatibility(@NonNull String googleAdsVe
         }
     }
 
+    /**
+     * Sets full valid URL for the /status endpoint of the PBS.
+     * Request to /status is sent when you call {@link PrebidMobile#initializeSdk(Context, SdkInitializationListener)}.
+     *
+     * @see <a href="https://docs.prebid.org/prebid-server/endpoints/pbs-endpoint-status.html">GET /status</a>
+     */
+    public static void setCustomStatusEndpoint(String url) {
+        if (url != null) {
+            if (url.toLowerCase().startsWith("http") && Patterns.WEB_URL.matcher(url).matches()) {
```

Comment:
This check doesn't look accurate. 

`prebid.org` is also valid URL, or `www.prebid.org` or some IP address. But this check won't allow such endpoints. 

Can we use `URLUtil.isValidUrl(url)`?